.github/workflows/ci.yaml

```
@@ -42,8 +46,7 @@ jobs:
           # Bazelisk will download bazel to here, ensure it is cached between runs.
           XDG_CACHE_HOME: ~/.cache/bazel-repo
         run: |
-          bazel --bazelrc=.github/workflows/ci.bazelrc --bazelrc=.bazelrc build //example/js:image //example/py:image
-          docker load -i bazel-bin/example/js/image.tar
-          docker run --rm image:latest
-          docker load -i bazel-bin/example/py/image.tar
-          docker run --rm image:latest
+          bazel --bazelrc=.github/workflows/ci.bazelrc --bazelrc=.bazelrc build //example/js:image
+          skopeo copy oci:bazel-bin/example/js/image docker-archive:js_tarball.tar --additional-tag example/js:latest
+          docker load -i js_tarball.tar
```

Always use long-format flags on scripts.
```suggestion
          docker load --input js_tarball.tar
```